src/components/shared/form_elements/RadioFieldTooltip.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<div class="radio-field-tooltip">
+		<InfoIcon/>
+		<div class="radio-field-tooltip-text">
+			<slot/>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+
+import InfoIcon from '@src/components/shared/icons/InfoIcon.vue';
+
+</script>
+
+<style scoped lang="scss">
+@use '@src/scss/settings/colors';
+
+.radio-field-tooltip {
+	position: absolute;
+	right: 10px;
+	top: 50%;
+	margin-top: -8px;
+	height: 16px;
+	width: 17px;
+
+	&-text {
+		position: absolute;
+		visibility: hidden;
+		opacity: 0;
+		height: 60px;
```

Comment:
maybe we could give it a `height: fit-content`,
but then a new positioning has to be defined

Reply:
Lets just make it wider for now and if we need to in the future we can add dynamic width calculation